### PR TITLE
fix: error visibility after clicking on min/max

### PR DESCRIPTION
**Pull request by mstrasinskis**
# Motivation

Solves the remaining error message after clicking on min/max button.
Steps:
- Add a too big number for the dissolve delay.
- Click on the Min/Max
- Input updated to a valid value, but the error is still displayed.

# Changes

- update error visibility after min/max click

# Tests

- "should update error message after Min/Max click"

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

File: frontend/src/tests/lib/components/ui/DayInput.spec.ts [hunk 1]
```diff
@@ -143,7 +143,36 @@ describe("DayInput", () => {
     expect(inputElement.value).toBe("30");
   });
 
-  it("should take into accout the max when rounding up", async () => {
+  it("should update error message after Min/Max click", async () => {
```
Comment (lmuntaner) [inline frontend/src/tests/lib/components/ui/DayInput.spec.ts:146]: Could you add a test also in SetDissolveDelay. That would be more accurate to how the user saw the error.

Comment (mstrasinskis) [reply frontend/src/tests/lib/components/ui/DayInput.spec.ts:146]: Thanks, updated.

**Review by lmuntaner — COMMENTED** [review]
Let me know when it's ready.

File: frontend/src/tests/lib/components/ui/DayInput.spec.ts (hunk 1 above)
Comment (mstrasinskis) [reply frontend/src/tests/lib/components/ui/DayInput.spec.ts:146]: Could you please take a look once again?

**Review by lmuntaner — APPROVED** [review]
LGTM! Thanks!

**Pull request merged**